src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/X500NameEncoder.cs

```diff
@@ -26,6 +30,11 @@ internal static class X500NameEncoder
             ';',
         };
 
+        private static readonly List<char> s_useSemicolonSeparators = new List<char>(1) { ';' };
+        private static readonly List<char> s_useCommaSeparators = new List<char>(1) { ',' };
+        private static readonly List<char> s_useNewlineSeparators = new List<char>(2) { '\r', '\n' };
+        private static readonly List<char> s_defaultSeparators = new List<char>(2) { ',', ';' };
```

Comment:
Mainly just that I thought that calling `separators.Contains(c)` looked a lot better than `separators.IndexOf(c) >= 0`; and went with "reads better" since there's only ever the 4.